src/managers/TenorManager.ts

```diff
@@ -39,7 +38,7 @@ class TenorManager {
 			'key': this.apiKey,
 			'client_key': this.clientKey,
 			'contentfilter': this.contentFilter,
-			'media_filter': MEDIA_FILTER,
+			'media_filter': MediaFilter.BASIC,
```

Comment:
This is not a valid media filter

![image](https://user-images.githubusercontent.com/23432278/211884386-1e1daa99-7f12-4bfa-96b7-f64f40f3a354.png)

https://developers.google.com/tenor/guides/endpoints

Reply:
The api accepts basic and minimal this and returns the different formats.

Reply:
Using undocumented api is not good idea

Reply:
It's still documented here: https://tenor.com/gifapi/documentation#endpoints-search

Reply:
Updated.